taskset.go

```diff
@@ -185,7 +186,9 @@ func (ts *TaskSet) MustUpdateTask(task Task) {
 		task.Resolved = time.Now()
 	}
 
-	task.WritePending = true
+	if !task.Deleted {
```

Comment:
I think it's best to decide whether to call `SaveToDisk()` in `SavePendingChanges()` rather than doing this early, as in theory the task could be mutated again before the SavePendingChanges.

Reply:
So, no special handling of deleted tasks in `MustUpdateTask`, then?